### DISCUSSION TITLE
macOS升级后不能使用V2rayU解决方案找到了。

### DISCUSSION THREAD

**Discussion by PhersonYu** [yanue/V2rayU#1248] [Ideas]
直接下载小火箭，M芯片的可以直接在appstore下载手机或pad软件。clash实在太慢了，v2rayU只能等软件更新了。小火箭亲测成功，体验和v2rayU一样快。

**Comment by ZZY1970** [comment]
M2确实有个优势，就是可以使用iPhone或iPad应用，我也测试了小火箭。小火箭可以运行，但访问openai的web界面https://platform.openai.com/playground时，会发生白屏，无法工作，唯有v2rayU可以正常访问。
所以，用小火箭临时凑合是可以的，坐等开发者是正解。

**Comment by PhersonYu** [comment]
[image: image.png]
完全可以打开。

ZZY1970 ***@***.***> 于2023年7月8日周六 20:08写道：
